react/features/base/participants/functions.js

```diff
@@ -456,44 +456,65 @@ async function _getFirstLoadableAvatarUrl(participant, store) {
 }
 
 /**
- * Selector for retrieving ids of participants in the order that they are displayed in the filmstrip (with the
```

Comment:
This is used also to sort the tile view. Do we want to change it to this? I thought @jallamsetty1 's work on this was to keep it that way.

Reply:
The tile view seems to have its own function for this, and this function is only used in the participant list. And we decided on a different ordering for the participant list, that's why this PR reverts the changes made by Jaya

Reply:
@robertpin I still don't understand why this is being reverted. This calculation is done on every render which is very inefficient especially when you have large number of participants. As part of the performance optimizations for large calls we have tried to minimize operations on arrays and switched to Sets since insertion and deletion on Sets is O(1). We already have a sorted list in redux and this matches with the current order of the thumbnails in the filmstrip. I think it is very bad idea to revert this.

Reply:
Ok, then for now I'll leave it as it was, but the change is still needed because we want to simplify the ordering the participants list and having the same order as the thumbnails in the filmstrip isn't necessary. The ordering we decided on is:
1. Dominant speaker
2. Raised hands
3. The rest, sorted alphabetically
This ordering makes is easier to find people in the participants list and is easy to understand. I'll try to optimize it and introduce it in another PR

Reply:
Makes sense to keep it in a different PR indeed.

Reply:
Once reordering is enabled, this is how the list is mostly going to look like except for the raised hand participants. They are first in the list, followed by the active speakers (sorted alphabetically) and the rest of the sorted alphabetically.